### DISCUSSION TITLE
Help upgrading to latest/

### DISCUSSION THREAD

**Discussion by backspaces** [ankitskvmdam/clean-jsdoc-theme#221] [General]
I'm back on version 4.1.8, and lost track of the progress in clean-jsdoc-theme.
Also you had helped me do things like move the jsdoc.js file to a config folder and a few other great things like copying tutorials. Oh, btw, I've pretty much migrated to deno for simple scripts, its so much more "browser-like" JS than node.

Do you have any suggestions on how to upgrade? My bet it's simpler now!

Thanks! Looks great.

**Comment by ankitskvmdam** [comment]
@backspaces Yes I remembered.

I think there won't be any issue in moving from 4.1.8 to 4.2.9, It is just we start using `var` in css, meaning very very old browsers are now not supported (Ref: https://caniuse.com/css-variables).

Apart from this there are few bug fixes in the search component.

I don't have any experience using deno, but if you find any bug/issue, feel free to share, I will try to find a solution.

**Comment by backspaces** [comment]
The new repo looks really great even tho a bit exotic/sophisticated!  Thanks!

Some questions: 
  - Which version of jsdoc do you use!?  Your package.json has  "@jsdoc/salty": "^0.2.4" ?? And the jsdoc github has a  - JSDoc 3.5.5 Sep 13, 2017!! The NPM page shows 4.0.2! We used 3.6.10 when I first used clean-jsdoc-theme.
  - I'll start by putting jsdoc.json in my root, but would it be OK to move it to my config/ folder having all the other configs my project uses?
  - Would it be best to just move my existing  docs off to the side and start from scratch? Seems enough changes have occurred that I'd run into trouble migrating piece-meal.
  - Could I just copy your jsdoc.json for now? Or would that run into trouble. I do have tutorials that I need to manage. And the sections would likely need fiddling with.

Sorry to be behind but I'm ready and willing! Thanks again.

**Reply by ankitskvmdam** [reply]
Hi,
Sorry for the late reply.

1. It is fine to use the latest version of JSDoc. I am using the latest version of JSDoc.
2. Not 100% sure, but I think it would be fine to move the config file into config folder.
3. There are not much changes in terms of behaviour, although as a backup you can keep a copy of your existing docs. 
4. I think you can stay with your jsdoc.json.

Hope this help.

**Comment by ankitskvmdam** [comment]
v5 is released, with a full migration guide: https://ankdev.me/clean-jsdoc-theme/theme/migrate-v4-to-v5/ Closing this. Please open a new issue if you hit a specific problem upgrading.
